docs/src/pages/components/slider/slider.md

```diff
@@ -77,6 +77,16 @@ Here are some examples of customizing the component. You can learn more about th
 
 {{"demo": "pages/components/slider/VerticalSlider.js"}}
 
+**WARNING**: Chrome, Safari and newer Edge versions i.e. any browser based on WebKit exposes `<Slider orientation="vertical" />` as horizontal ([chromium issue #1158217](https://bugs.chromium.org/p/chromium/issues/detail?id=1158217)).
+By applying `-webkit-appearance: slider-vertical;` the slider is exposed as vertical.
+
+However, by applying `-webkit-appearance: slider-vertical;` keyboard navigation for horizontal keys (`ArrowLeft`, `ArrowRight`) is reversed ([chromium issue #1162640](https://bugs.chromium.org/p/chromium/issues/detail?id=1162640)).
+Usually, up and right should increase and left and down should decrease the value.
+If you apply `-webkit-appearance` you could prevent keyboard navigation for horizontal arrow keys for a truly vertical slider.
+This might be less confusing to users compared to a change in direction.
```

Comment:
@mbrookes Could you take a look at the wording if you got time?

Reply:
What do you think about moving this new section under `## Accessibility`?

Reply:
I added it to the vertical section on purpose because it's directly related to it. `Accessibility` is a general section or rather an umbrella section for everything that doesn't fit into other sections.